pages/Add/index.js

```diff
@@ -27,16 +16,11 @@ export default function Add() {
       fishweight: +e.target.weight.value,
       fishlength: +e.target.length.value,
       location: e.target.location.value,
-      date: startDate,
+      date: startDate.toString(),
```

Comment:
You could use the .**toISOString()** method to be safe. It's the international standard for handling date and time data. Your app is probably not going to experience any issues if you don't use it, but now at least you've heard of it =)

Reply:
i didn't know that, thanks :)